platform-includes/performance/custom-sampling-context/node.mdx

```diff
@@ -1,22 +1,28 @@
 ```javascript
-Sentry.startTransaction(
+Sentry.startSpan(
   {
-    // `transactionContext` - will be recorded on transaction
     name: 'GET /search',
     op: 'search',
-    data: {
-      queryParams: {
-        animal: "dog",
-        type: "very good"
-      }
+    attributes: {
+      queryForAnimal: "dog",
+      queryForType: "very good"
```

Comment:
l: No strong opinion but should we promote a more formalized way of setting attribute keys? Something like

```suggestion
      "queryParam.animal": "dog",
      "queryParam.type": "very good"
```

Which more directly represents the flat attributes Record style?

Reply:
makes sense to me!